war/src/main/webapp/help/system-config/systemMessage.html

```diff
@@ -1,5 +1,6 @@
 <div>
-  This message will be displayed on <a href=".">the top page</a>.
-  Useful for posting a system-wide notification to users.
-  Can contain HTML tags or whatever markup language is defined for the system.
+  This message will be displayed at the top of the
```

Comment:
@jtnord Admin monitors are not shown anywhere but `/manage`. Something we should address in a way similar to https://github.com/daniel-beck/jenkins-visible-administrative-monitors-plugin

FWIW this is shown on the top of every *view*, not just the index page.